### ISSUE TITLE
linux support 

### ISSUE THREAD

**Issue by dev-ahmed** [jhen0409/react-native-debugger#85]
** Please ensure:

- [ ] I'm using the latest version of react-native-debugger
- [ ] I have read [README.md](https://github.com/jhen0409/react-native-debugger/blob/master/README.md)

** Please provide the following information if you found some bugs or have question:

React Native version: [FILL THIS OUT]
Platform: [FILL THIS OUT: iOS, Android, ...]
Is real device of platform: [Yes or No]
Operating System: [FILL THIS OUT: macOS, Linux, or Windows?]

**Comment by jhen0409** [comment]
Hi, we have `rn-debugger-linux-x64.zip` in the [releases](https://github.com/jhen0409/react-native-debugger/releases) page, or have you got any problems about use the Linux binary? Need more information.

**Comment by jhen0409** [comment]
> i couldn't find any installation command for linux.

We haven't a command yet (like `apt`) for install the Linux binary, if anyone could contributing it I'll happy to accept.

**Comment by dev-ahmed** [comment]
yes it works with me but i must open the debugger manually , so any workaround to open the debugger automatically like mac ?

**Comment by juanlet** [comment]
Just create a bash file to execute it, put it on usr/local/bin and give it the name of rn-debug and give execution permissions with chmod +x rn-debug ...reset the console and after this you can run the debugger just with the rn-debug comand

**Comment by chaicai333** [comment]
@juanlet , hi can you tell on more detail how to create the bash file ?

**Comment by danilobuerger** [comment]
To create a deb, it would really be easy with https://github.com/unindented/electron-installer-debian
Just
```
electron-installer-debian  --src build --dest dist --arch amd64
```
on the current build output

**Comment by alpredoRuben** [comment]
How to install rn-debugger-linux-x64.zip on linux elementary?

**Comment by StartupSam** [comment]
Really unsure how to proceed with installation on this. I unzipped the file and attempted to open the executable, but nothing happens (a silent exit I think?). Not sure if this is an issue or if I'm doing something wrong.

**Comment by pjivers-bom** [comment]
On the command line, navigate to where you extracted the package and execute it with `./react-native-debugger`. You can also add an alias for it in your `~/.bashrc`/`~/.bash_profile` (or `~/.zshrc` if you're using zsh), e.g.
```
alias rn-debug="~/bin/react-native-debugger/react-native-debugger"
```
